CMakeLists.txt

```diff
@@ -158,6 +158,10 @@ set(CP2K_BUILD_OPTIONS
 set_property(CACHE CP2K_BUILD_OPTIONS PROPERTY STRINGS
                                                ${CP2K_BUILD_OPTIONS_LIST})
 
+set(CP2K_DATA_DIR
+    "default"                                                                                                                                                 |
+    CACHE STRING "Set the location for cp2k data")                                                                                                            |
```

Comment:
These `|` characters seem out of place.

Reply:
That was weird, not sure where those came from